style/_jieun.scss

```
@@ -1,3 +1,83 @@
 .three {
-  background: blue;
+  // background: rgba(203, 203, 255, 0.723);
+
+  border: 1px solid #aaa;
+  border-radius: 5px;
 }
+
+.news-company {
+  padding-top : 20px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  article {
+    width: 200px;
+    height: 50px;
+    list-style: none;
+    border: 1px solid #aaa;
+    text-align: center;
+    a {
+      display: block;
+      line-height: 50px;
+    }
+  }
+}
```

```suggestion
.news-company {
  margin: 18px 20px;
  display: flex;
  flex-wrap: wrap;

  border: solid #ebebeb;
  border-width: 1px 0 0 1px;
  border-radius: 4px;
  article {
    text-align: center;
    flex-grow: 1;
    padding: 18px 0;

    width: 15.66%;
    border: solid #ebebeb;
    border-width: 0 1px 1px 0;

    &:nth-child(6) {
      border-radius: 0 4px 0 0;
    }
    &:nth-child(7) {
      border-radius: 0 0 0 4px;
    }
    &:last-child {
      border-radius: 0 0 4px 0;
    }
  }
}
```
border가 겹치지 않게 하기위해서 변화를 줬지만 재 사용성이 떨어져서 좋은 방법인거 같진 않아요 :(
네이버는 float를 사용했는데.. 저희는 여기서 hover 했을 때 변화 되는거를 추가하는게 좋을꺼 같아요!